src/app/pages/RewardPage/components/FeesEarnedTab/index.tsx

```diff
@@ -91,9 +93,10 @@ export const FeesEarnedTab: React.FC<IFeesEarnedTabProps> = ({
           color={RewardsDetailColor.Yellow}
           title={t(translations.rewardPage.fee.stakingFee)}
           availableAmount={weiTo18(amountToClaim)}
-          totalEarnedAmount={weiTo18(totalRewardsEarned)}
+          totalEarnedAmount={bignumber(totalStakingFees)
+            .add(weiTo18(amountToClaim))
+            .toString()}
```

Comment:
```suggestion
          totalEarnedAmount={weiTo18(
            bignumber(totalStakingFees).add(amountToClaim),
          )}
```

Reply:
the totalStakingFees is already converted from WEI and in this case, it will return the totalStakingFees without adding amountToClaim, i will add changes in RewardDetails regarding this

Reply:
Nice solution, thanks :)